src/backend/InvenTree/stock/migrations/0065_auto_20210701_0509.py

```diff
@@ -4,22 +4,6 @@
 from django.db import migrations
 import djmoney.models.fields
 
-from django.db.migrations.recorder import MigrationRecorder
-
-
-def show_migrations(apps, schema_editor):
-    """Show the latest migrations from each app"""
-
-    for app in apps.get_app_configs():
-
-        label = app.label
-
-        migrations = MigrationRecorder.Migration.objects.filter(app=app).order_by('-applied')[:5]
-
-        print(f"{label} migrations:")
-        for m in migrations:
-            print(f" - {m.name}")
-
```

Comment:
@SchrodingersGat I would remove this as I do not see a business value in it. Do you think it needs to be kept?

Reply:
@matmair I added this to assist with user debugging - there have been a few cases where I have needed to see the state of a particular app, and having `invoke.showmigrations part` is far simpler to accomplish. I think it has its use

Reply:
But was has that to do with running the command on every instance through migrations (one time)

Reply:
@matmair oh, sorry I totally misread the context of this code. Yeah, that looks like an old debug hook. In fact if you look down further in the file, that code is not even getting called!

Reply:
That can 100% be removed